dataStandard_versioned_docs/version-5/model-reference/intervention-domain/entities-references-and-descriptors.md

```diff
@@ -0,0 +1,64 @@
+# Intervention Domain - Entities, References, and Descriptors
+
+## Intervention Domain Entities
+
+| Name | Description |
+| --- | --- |
+| EducationOrganizationInterventionPrescriptionAssociation | This association indicates interventions made available by an education organization. Often, a district-level education organization purchases a set of intervention prescriptions and makes them available to its schools for use on demand. |
```

Comment:
This name is causing the table to extend pretty far. Not sure we can do anything about it but just noting.

Reply:
Mustafa and I have discussed needing to review these tables in the near future, to see if we can find a better way of displaying them.